### ISSUE TITLE
Build.bat fails if only VS2015 is installed

### ISSUE THREAD

**Issue by bkoelman** [dennisdoomen/CSharpGuidelines#50]
On my machine, the registry key `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\MSBuild\ToolsVersions\12.0` does not exist.

This causes Build.bat to fail:

		Get-ItemProperty : Cannot find path
		'HKLM:\SOFTWARE\Microsoft\MSBuild\ToolsVersions\12.0' because it does not
		exist.
		At E:\Bart\Source\Repos\CSharpGuidelines\Build\psake.psm1:625 char:61
		+         $frameworkDirs = @($buildToolsVersions | foreach { (Get-ItemProperty
		-Pa ...
		+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
		~~~
		    + CategoryInfo          : ObjectNotFound: (HKLM:\SOFTWARE\...lsVersions\12
		   .0:String) [Get-ItemProperty], ItemNotFoundException
		    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetIt
		   emPropertyCommand

		Error: 10/18/2016 10:14:24 PM:
		At E:\Bart\Source\Repos\CSharpGuidelines\Build\psake.psm1:639 char:22 +     $fr
		ameworkDirs | foreach { Assert (test-path $_ -pathType Container) ($msgs. ... +
		 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
		~~ [<<==>>] Exception: Cannot bind argument to parameter 'Path' because it is n
		ull.

Chaning "12.0" to "14.0" at line https://github.com/dennisdoomen/CSharpGuidelines/blob/master/Build/psake.psm1#L584 fixes the issue for me.

**Comment by dennisdoomen** [comment]
I need to upgrade PSake. The one I used doesn't official support Visual Studio 2015. My machine apparently does have the build tools.

**Comment by dennisdoomen** [comment]
Fixed by #53
